token_handler.js

```diff
@@ -139,6 +140,8 @@ module.exports = (req, res) => {
         // If the jku header is whitelisted, create a set of potential keys
         // by dereferencing the jku URL
         if (header.jku && clientDetailsToken.jwks_url) {
+            console.log("jwks url: " + clientDetailsToken.jwks_url);
```

Comment:
I'd want to remove this to simply keep the diff count low.  But maybe we can do that later if this is useful now still.

Reply:
change made